README.md

```diff
@@ -2,127 +2,84 @@
 
 ## Description
 
-VTEX App that shows a collection of products.
+VTEX App that shows a collection of products. This is a VTEX app that is used by Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [0.x]    | **Maintenance LTS** |  2018-04-19     | 2018-11-08            | March 2019  | 1.x
 | [1.x]    | **Current Release** |  2018-11-08     |                       |             | 2.x
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
 
-## Continuous Integrations
-
-### Travis CI
+## Table of Contents
+- [Usage](#usage)
+- [API](#api)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
 
-[![Build Status](https://travis-ci.org/vtex-apps/shelf.svg?branch=master)](https://travis-ci.org/vtex-apps/shelf)
 
 ## Usage
+This app use our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-> 1- Add the dependency in your `manifest.json`
+To use this app you need to import in your dependencies on `manifest.json`.
```

Comment:
Correct: 
`To use this app you need to import it in your dependencies on the `manifest.json` file.

Reply:
I think this is a nit, on manifest.json seems correct too.